workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/DealRegistrationList/DealRegistrationList.tsx

```diff
@@ -32,6 +32,12 @@ import useGetListItemsFromDealRegistration from './hooks/useGetListItemsFromDeal
 export type DealRegistrationItem = {
 	[key in DealRegistrationColumnKey]?: any;
 };
+
+type TableColumn = {
+	columnKey: DealRegistrationColumnKey;
+	label: string | JSX.Element;
+	size?: 'sm' | 'md';
+};
```

Comment:
```suggestion
```
Why are you creating a new TableColumn? We already have a interface defined for this type.